sepal_ui/frontend/styles.py

```diff
@@ -93,3 +141,24 @@ class Styles(v.VuetifyTemplate):
     "DEFAULT": _other,
     "PARENT": _parent,
 }
+
+
+# Set default styles to layers
+
+layer_style = {  # default styling of the layer
```

Comment:
could you change the comments (I think they are coming from the alert module and they make no sens in this file)